src/diva.js

```diff
@@ -62,9 +89,16 @@ function updateQRContentWithApiEndpoint(qrContent) {
   };
 }
 
+function attributesToContent(attributes, attributesLabel) {
```

Comment:
I think we should just require a syntax similar to what the `content` format requires.
We can then also supply convenience methods for simple cases.

The alternative is that we suplly a kind of DSL/builder syntax but that seems overkill to me.

Reply:
will do in DIVA-58